lib/activerecord_batch_update.rb

```diff
@@ -35,6 +35,7 @@ def batch_update(entries, columns:, batch_size: 100, validate: true)
       end
 
       connection.clear_query_cache if connection.query_cache_enabled
+      entries.each(&:changes_applied)
```

Comment:
is there a way you can only apply the changes on the whitelist of columns passed at the top-level?